Fails to build Docker image on Windows from the Dockerfile

While buiding Docker image on Windows I found that I cannot build it.

From the directory with Dockerfile and the ASReview app I run this:
`docker build -t asreview/asreview .`

Which gets me to this error:

8 279.1 Specify configs in the ini-formatted file:
 279.1     /root/.npmrc
 279.1 or on the command line via: npm <command> --key=value
 279.1
 279.1 More configuration info: npm help config
 279.1 Configuration fields: npm help 7 config
 279.1
 279.1 npm@7.5.2 /usr/share/nodejs/npm
 279.1
 279.1
 279.1 Did you mean one of these?
 279.1     install
 279.1     uninstall
 279.4 npm ERR! code ENOENT
 279.4 npm ERR! syscall open
 279.4 npm ERR! path /app/package.json
 279.4 npm ERR! errno -2
 279.4 npm ERR! enoent ENOENT: no such file or directory, open '/app/package.json'
 279.4 npm ERR! enoent This is related to npm not being able to find a file.
 279.4 npm ERR! enoent
 279.4
 279.4 npm ERR! A complete log of this run can be found in:
 279.4 npm ERR!     /root/.npm/_logs/2023-05-18T11_43_18_541Z-debug.log
 279.4 Traceback (most recent call last):
 279.4   File "setup.py", line 83, in <module>
 279.4     setup(
 279.4   File "/usr/local/lib/python3.8/site-packages/setuptools/__init__.py", line 107, in setup
 279.4     return distutils.core.setup(**attrs)
 279.4   File "/usr/local/lib/python3.8/site-packages/setuptools/_distutils/core.py", line 185, in setup
 279.4     return run_commands(dist)
 279.4   File "/usr/local/lib/python3.8/site-packages/setuptools/_distutils/core.py", line 201, in run_commands
 279.4     dist.run_commands()
 279.4   File "/usr/local/lib/python3.8/site-packages/setuptools/_distutils/dist.py", line 969, in run_commands
 279.4     self.run_command(cmd)
 279.4   File "/usr/local/lib/python3.8/site-packages/setuptools/dist.py", line 1244, in run_command
 279.4     super().run_command(command)
 279.4   File "/usr/local/lib/python3.8/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
 279.4     cmd_obj.run()
 279.4   File "setup.py", line 74, in run
 279.4     subprocess.check_call("sh ./asreview/webapp/compile_assets.sh", shell=True)
 279.4   File "/usr/local/lib/python3.8/subprocess.py", line 364, in check_call
 279.4     raise CalledProcessError(retcode, cmd)
 279.4 subprocess.CalledProcessError: Command 'sh ./asreview/webapp/compile_assets.sh' returned non-zero exit status 254.

executor failed running [/bin/sh -c apt-get update     && apt-get install -y git npm     && pip3 install --upgrade pip setuptools     && python3 setup.py compile_assets     && pip3 install --user .     && pip3 install --user asreview-datatools asreview-insights asreview-makita asreview-wordcloud]: exit code: 1

**Version information**
 -  Windows
 - ASReview version: e.g. 0.6.2
 - Docker version 23.0.5 

**Additional context**
When I was building the images the same way on Mac and Linux, everything went smoothly

--- a/asreview/webapp/compile_assets.sh
+++ b/asreview/webapp/compile_assets.sh
@@ -1,4 +1,0 @@
-cd asreview/webapp/
-rm -rf build
-npm install
-npm run-script build

--- a/setup.py
+++ b/setup.py
@@ -19,6 +19,7 @@ import re
 import subprocess
 from io import open
 from os import path
+from pathlib import Path
 
 from setuptools import Command
 from setuptools import find_packages
@@ -71,7 +72,11 @@ class CompileAssets(Command):
 
     def run(self):
         """Run a command to compile and build assets."""
-        subprocess.check_call("sh ./asreview/webapp/compile_assets.sh", shell=True)
+
+        path_webapp = Path(__file__).parent / "asreview" / "webapp"
+
+        subprocess.check_call(["npm", "install"], cwd=str(path_webapp))
+        subprocess.check_call(["npm", "run-script", "build"], cwd=str(path_webapp))
 
 
 def get_cmdclass():

